artifacts/pom.xml

```diff
@@ -135,16 +135,40 @@
 		<!-- latest mock-sdk -->
 		<mock-sdk-latest.version>1.2.0.2</mock-sdk-latest.version>
 
+		<!-- certify sunbird plugin -->
 		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
-		<!-- certify sunbird plugin  -->
-		<certify-sunbird-plugin.version>0.2.1</certify-sunbird-plugin.version>
+		<certify-sunbird-plugin.version>0.3.0-SNAPSHOT</certify-sunbird-plugin.version>
 		<certify-sunbird-plugin.fileName>certify-sunbird-plugin.jar</certify-sunbird-plugin.fileName>
 		<!-- certify mosip identity plugin -->
-		<certify-mosip-identity-plugin.version>0.2.1</certify-mosip-identity-plugin.version>
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mosip-identity-plugin.version>0.3.0-SNAPSHOT</certify-mosip-identity-plugin.version>
 		<certify-mosip-identity-plugin.fileName>certify-mosip-identity-plugin.jar</certify-mosip-identity-plugin.fileName>
-		<!-- certify mock plugin -->
-		<certify-mock-plugin.version>0.2.1</certify-mock-plugin.version>
+        <!-- certify mock plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mock-plugin.version>0.3.0-SNAPSHOT</certify-mock-plugin.version>
 		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
+
+		<!-- certify mock ida dataprovider plugin -->
+		<certify-mock-ida-dataprovider-plugin.version>0.3.0-SNAPSHOT</certify-mock-ida-dataprovider-plugin.version>
+        <certify-mock-ida-dataprovider-plugin.fileName>certify-mock-ida-dataprovider-plugin.jar</certify-mock-ida-dataprovider-plugin.fileName>
+
+		<!-- certify postgres plugin -->
+		<certify-postgres-dataprovider-plugin.version>0.3.0-SNAPSHOT</certify-postgres-dataprovider-plugin.version>
+		<certify-postgres-dataprovider-plugin.fileName>certify-postgres-dataprovider-plugin.jar</certify-postgres-dataprovider-plugin.fileName>
+
+
+		<!-- esignet IDA wrapper -->
+		<esignet-ida-wrapper.version>1.2.1-SNAPSHOT</esignet-ida-wrapper.version>
+		<esignet-ida-wrapper.fileName>esignet-ida-wrapper.jar</esignet-ida-wrapper.fileName>
+		<esignet-digital-credential-wrapper.version>0.3.0-SNAPSHOT</esignet-digital-credential-wrapper.version>
+		<esignet-digital-credential-wrapper.fileName>sunbird-rc-esignet-integration-impl.jar</esignet-digital-credential-wrapper.fileName>
```

Comment:
why we need this eSignet releated entries ?

Reply:
Removed esignet entries